bql/semantic/semantic_test.go

```diff
@@ -96,6 +97,132 @@ func TestStatementAddData(t *testing.T) {
 	}
 }
 
+func TestGraphClauseString(t *testing.T) {
+	timeObj1 := time.Date(2019, 11, 20, 2, 30, 10, 5, time.UTC)
+	timeObj2 := time.Date(2019, 12, 3, 5, 40, 20, 7, time.UTC)
+	timeObj3 := time.Date(2019, 11, 30, 22, 10, 50, 3, time.UTC)
+	timeObj4 := time.Date(2019, 12, 2, 17, 0, 10, 15, time.UTC)
+	// Testing that NewNodeFromStrings is not the point of this package. Taking the example from the unit tests.
+	n, _ := node.NewNodeFromStrings("/some/type", "id_1")
+	// Testing NewImmutable is not the point of this package.
+	immutFoo, _ := predicate.NewImmutable("foo")
+	nO, _ := node.NewNodeFromStrings("/some/other/type", "id_2")
+	o := triple.NewNodeObject(nO)
+	table := []struct {
+		gc   *GraphClause
+		want string
+	}{
```

Comment:
For consistency with the rest of the files, would it be possible to explicitly name the fields instead of relying on position?

Reply:
Good catch! Done.